docker-compose.yml

```diff
@@ -20,8 +24,32 @@ services:
     networks:
       - backend
 
+  mysql:
+    image: mysql:8.0.21
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      - MYSQL_ROOT_PASSWORD=natlas-dev-password-do-not-use
+      - MYSQL_DATABASE=natlas
+    networks:
+      - backend
```

Comment:
Did we configure natlas-server to use this this anywhere?

Reply:
I didn't include it in here because I wasn't sure whether we wanted to change it as the default for the dev stack. I wanted the container definition in here to make it easier to use, but don't care one way or the other if we make it the default for dev.